tests/test_resolutions.py

```diff
@@ -240,7 +240,9 @@ def test_description(self) -> None:
 
         ctt = ContainsTheText(test_text)
 
-        expected_description = "Containing the text 'Wenn ist das Nunstück git und Slotermeyer?'."
+        expected_description = (
```

Comment:
Somehow we missed running black against this file?

Reply:
I think maybe we had a few PRs merged at the same time, one of which turned on `black` and others which added this stuff? I find it odd that `black` would only have two things to say about this file.